docs/README.md

```diff
@@ -46,6 +54,15 @@ Install ty into your current Python environment with pip:
 pip install ty
 ```
 
+### Running ty
```

Comment:
I'm not sure if this belongs in the "Installation" section.

Reply:
@zanieb Indeed; I have now changed that header appropriately to "Installation and basic usage".